Samples/LowLevelKernelCompilation/Program.cs

```diff
@@ -88,31 +88,28 @@ static void CompileAndLaunchKernel(Accelerator accelerator, int groupSize)
             // -------------------------------------------------------------------------------
             // Load the explicitly grouped kernel
             // Note that the kernel has to be disposed manually.
-            using (var kernel = accelerator.LoadKernel(compiledKernel))
-            {
-                var launcher = kernel.CreateLauncherDelegate<Action<AcceleratorStream, KernelConfig, ArrayView<int>, int>>();
-                // -------------------------------------------------------------------------------
+            using var kernel = accelerator.LoadKernel(compiledKernel);
+            var launcher = kernel.CreateLauncherDelegate<Action<AcceleratorStream, KernelConfig, ArrayView<int>, int>>();
+            // -------------------------------------------------------------------------------
 
-                using (var buffer = accelerator.Allocate1D<int>(1024))
-                {
-                    // You can also use kernel.Launch; however, the generic launch method involves boxing.
-                    launcher(
-                        accelerator.DefaultStream,
-                        (((int)buffer.Length + groupSize - 1) / groupSize, // Compute the number of groups (round up)
-                         groupSize),                                       // Use the given group size
-                        buffer.View,
-                        42);
-
-                    accelerator.Synchronize();
-
-                    // Resolve and verify data
-                    var data = buffer.GetAsArray1D();
-                    for (int i = 0, e = data.Length; i < e; ++i)
-                    {
-                        if (data[i] != 42 + i)
-                            Console.WriteLine($"Error at element location {i}: {data[i]} found");
-                    }
-                }
+            using var buffer = accelerator.Allocate1D<int>(1024);
+
+            // You can also use kernel.Launch; however, the generic launch method involves boxing.
+            launcher(
+                accelerator.DefaultStream,
+                (((int)buffer.Length + groupSize - 1) / groupSize, // Compute the number of groups (round up)
+                 groupSize),                                       // Use the given group size
+                buffer.View,
+                42);
+
+            accelerator.Synchronize();
```

Comment:
nit: should we remove the explicit synchronization here and place a note that the synchronization happens implicitly by a call to `GetAsArray1D()`?